test/plugins/_index.js

```diff
@@ -36,7 +36,7 @@ describe('plugins tests', function() {
                         plugins = {},
                         svgo;
 
-                    plugins[name] = (params) ? JSON.parse(params) : true;
+                    plugins[name] = (params) ? eval('var x = ' + params + '; x;') : true;
```

Comment:
This change was necessary to allow function definitions in the passed attributes. The addAriaLabels plugin usually auto-generates a random ID to link title/desc to `aria-labelledby`, but with static fixtures this does not work unless the random part is made static as well.